vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -423,29 +442,33 @@ private static void timingMessage(String message) {
         @Option(name = "-showrp", usage = "Show Runtime plan.")
         private boolean showRP;
 
-        @Option(name = "-compileonly", usage = "Compile the query and stop.")
-        private boolean compileOnly;
+        // Optional (Not supported by REST API) parameters. Only used for creating a local hyracks cluster
```

Comment:
Then lets remove these options. They should be defined in a server config (either local or on the server).